flags.go

```diff
@@ -38,22 +38,24 @@ func addFlags(log logger.Logger, flags FlagSet, o any) {
 			if !f.IsExported() {
 				continue
 			}
-			v := v.Field(i)
+			fv := v.Field(i)
```

Comment:
nit: I would leave this `v` unless you need to reference it below (which looks as though is not happening). I intentionally shadowed the containing value, it was too easy to get the wrong one mixed up when passing values around to reflect calls